salesforce-oms/1.1/modules/ROOT/pages/sfdc-oms-setup-customization-guide.adoc

```diff
@@ -10,12 +10,21 @@ Follow the instructions in the https://resources.docs.salesforce.com/latest/late
 * Disable the *Enable Reduction Orders* option in *Order Settings*.
 * Enable *Order Management*, *Use Person Accounts*, and *High Scale Orders*
 * Activate *Multiple Currencies*
+* Enable *State* and *Country/Territory* picklists
 * <<configure_payment_gateway>>
+* Setup a Digital Wallet or Credit Card payment method 
+* Associate Account with payment method
+* Create a new Digital Wallet from the Digital Wallets page if there is no Payment Method already configured for the Account. A Digital Wallet can be created by filling in the Account, Processing Mode, Nickname and Type fields. 
+
 
 == Customization
 
 Install the Salesforce package from the https://anypoint.mulesoft.com/exchange/997d5e99-287f-4f68-bc95-ed435d7c5797/accel-b2b-common-resources-src/[Accelerator B2B Common Resources] project and verify that the configuration is correct in your Salesforce organization.
 
+Once the package is installed successfully, proceed with the following steps:
+* Create matching Rule Account Interchange Code Match with Matching Cretaria Account: B2B_EDISenderCode EXACT MatchBlank = FALSE 
+* Create duplicate rule named 'Account Interchange Code Rule' with the matching rule 'Account Interchange Code Match'. Set the order to 1, giving it precedence over the standard account matching rule.
```

Comment:
I don't understand these steps. Where do you perform them?

Reply:
We need to create a matching rule named 'Account Interchange Code Match' with matching criteria "B2B_EDISenderCode EXACT MatchBlank = FALSE"

Reply:
these needs to done on the OMS Rule Setup

Reply:
![Screenshot 2024-06-12 at 2 07 41 PM](https://github.com/mulesoft/docs-accelerators/assets/48571056/6a4ff1db-aeac-4b9a-bf0d-59684a05612f) this is a common setup inside Salesforce and developers/admin knows it